scripts/sct_dmri_concat_b0_and_dwi.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# Merge b=0 and dMRI data and output appropriate bvecs/bvals files.
+#
+# Copyright (c) 2019 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Author: Julien Cohen-Adad
+#
+# About the license: see the file LICENSE.TXT
+
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+
+from dipy.data.fetcher import read_bvals_bvecs
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+from spinalcordtoolbox.image import Image, concat_data
+
+import sct_utils as sct
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description="Concatenate b=0 scans with DWI time series and update the bvecs and bvals files. Note that you can"
+                    "concatenate more than two files (e.g.: b0 dwi1 dw2 dw3).",
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        '-i',
+        help="Input 4d files, separated by space, listed in the right order of concatenation. Example: b0.nii dmri.nii",
+        nargs='+',
+        metavar=Metavar.file,
+        required=True,
+    )
+    mandatory.add_argument(
+        '-bval',
+        help="Bvals file(s). Example: bvals.txt",
+        nargs='+',
+        metavar=Metavar.file,
+        required=True,
+    )
+    mandatory.add_argument(
+        '-bvec',
+        help="Bvecs file(s). Example: bvecs.txt",
+        nargs='+',
+        metavar=Metavar.file,
+        required=True,
+    )
+    mandatory.add_argument(
+        '-order',
+        help="Order of b=0 and DWI files entered in flag '-i', separated by space. Example: b0 dwi",
```

Comment:
Just to make sure I understand correctly: If I had `b0.nii dmri1.nii dmri2.nii` as input, I would have to indicate `b0 dwi dwi` for `-order`, correct?

Reply:
yup! that"s correct